SpatialGDK/Source/SpatialGDKEditor/Private/SpatialTestSettings.cpp

```diff
@@ -62,13 +67,15 @@ void FSpatialTestSettings::Override(const FString& MapName)
 	}
 
 	// Generated config, applied to generated maps
-	FString GeneratedMapOverridesFilename =
-		GeneratedOverrideSettingsBaseFilename + FPackageName::GetShortName(MapName) + (OverrideSettingsFileExtension);
-	if (FPaths::FileExists(GeneratedMapOverridesFilename))
+	const FString GeneratedMapConfigurationFilename = GenerateMapConfigurationFilename(MapName);
+	if (FPaths::FileExists(GeneratedMapConfigurationFilename) == true)
```

Comment:
```suggestion
	if (FPaths::FileExists(GeneratedMapConfigurationFilename))
```

Reply:
Oh I thought this was a standard that we were suppose to use? Tbh I really hate writing == true :joy:

Reply:
For truthy (aka [implicitly](https://en.cppreference.com/w/cpp/language/implicit_conversion) casteable to bool such as integers) or classes with a [bool operator](https://en.cppreference.com/w/cpp/memory/shared_ptr/operator_bool) defined values its preferred by some to be explicit. E.g.
`if (SomeObject == nullptr)` over  `if (SomeObject)`
For booleans its fine to always
`if (SomeBool)`

Reply:
Ah this makes sense, oky will do, thanks for the clarifications :)